test/fixtures/basic/web.yml.erb

```diff
@@ -4,10 +4,10 @@ metadata:
   name: web
   labels:
     name: web
-    app: trashbin
+    app: basic
```

Comment:
This is usually just the name of the app itself by our conventions (e.g. the repo name).